pkgs/games/am2rlauncher/default.nix

```diff
@@ -0,0 +1,113 @@
+{ lib,
+  buildDotnetModule,
+  pkgsi686Linux,
+  glibc,
+  gtk3,
+  libappindicator,
+  webkitgtk,
+  e2fsprogs,
+  libnotify,
+  libgit2,
+  openssl,
+  xdelta,
+  file,
+  busybox,
+  openjdk,
+  steam-run,
+  patchelf,
+  fetchFromGitHub,
+  makeDesktopItem }:
+
+buildDotnetModule rec {
+  pname = "AM2RLauncher";
```

Comment:
I'd change the pname to `am2r-launcher` (same in `all-packages.nix`).
https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

Reply:
FWIW, i'd personally rather have `am2rlauncher` than `am2r-launcher`. That's also the package name on the AUR.